docs/operating/enhanced.md

```diff
@@ -13,9 +13,7 @@ import TabItem from '@theme/TabItem';
 
 ## Who Should Use This Guide
 
-This guide provides instructions on how to run a self-managed instance of Neon Proxy that offers more flexibility, configurability, and control for its Operator. It is designed for:
-* More **advanced** operators who would like to build their own Kubernetes clusters from scratch and run Neon Proxy on them
-* **Enterprise** operators with cluster solutions on-premise or with third-party public cloud providers such as AWS, Google Cloud, and Microsoft Azure
+This guide provides instructions on how to run a self-managed instance of Neon Proxy that offers more flexibility, configurability, and control for its Operator. It is designed for operators who use or would like to use a Kubernetes cluster (ether on-premise or third-party provided e.g. AWS(EKS), Azure(AKS), GCP(GKE)) for hosting Neon Proxy installation.
```

Comment:
```suggestion
This guide provides instructions on how to run a self-managed instance of Neon Proxy that offers more flexibility, configurability, and control for its Operator. It is designed for operators who use or would like to use a Kubernetes cluster (ether on-premise or third-party provided e.g. AWS(EKS), Azure(AKS), GCP(GKE)) for hosting Neon Proxy installation.
```
```suggestion
The guide provides instructions on how to run a self-managed instance of Neon Proxy that offers more flexibility, configurability, and control for its Operator. It is designed for operators who use or would like to use a Kubernetes cluster to host the Neon Proxy installation. This could be either on-premise or provided by a third party, such as AWS(EKS), Azure(AKS), or GCP(GKE).
```